### DISCUSSION TITLE
Disable or limit erlang dumps on RabbitMQ

### DISCUSSION THREAD

**Discussion by kulbalgis** [rabbitmq/rabbitmq-server#6774] [Questions] [answered]
Hello
I am facing issue when erlang dump is generated because erlang process run out of memory. Generated dump is big as RAM and it not only filling up /var/log/ mount but it is useless in general because explanation of a failure is in the first we lines of the dump. I tried to set erlang variable via rabbitmq-env.conf but couldn’t make it work. How can disable or limit erlang dumps?
ERL_CRASH_DUMP_BYTES

**Comment by michaelklishin** [comment, marked as answer]
Using `ERL_CRASH_DUMP_BYTES`. `rabbitmq-env.conf` likely filters out non-RabbitMQ related variables. Just set it in the env for the effective user any other way.

**Reply by kulbalgis** [reply]
Thak you for the reply Michael, looks like this did the trick.
`RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS='-env ERL_CRASH_DUMP_BYTES 0'`

**Comment by michaelklishin** [comment]
`ERL_CRASH_DUMP_SECONDS` set to `0` is another way of [disabling crush dumps](https://erlef.github.io/security-wg/secure_coding_and_deployment_hardening/crash_dumps).

**Comment by lukebakken** [comment]
It should work to set `ERL_CRASH_DUMP_BYTES` in `rabbitmq-env.conf`, but you would have to export it like this:

```
export ERL_CRASH_DUMP_BYTES=0
```

**Reply by michaelklishin** [reply]
I've added a small section on this to the [Runtime guide](https://rabbitmq.com/runtime.html).
